projects/schematics/src/migrations/4_0/constructor-deprecations/data/routing.service.migration.ts

```diff
@@ -20,5 +22,8 @@ export const ROUTING_SERVICE_MIGRATION: ConstructorDeprecation = {
     { className: SEMANTIC_PATH_SERVICE, importPath: SPARTACUS_CORE },
     { className: ROUTING_PARAMS_SERVICE, importPath: SPARTACUS_CORE },
   ],
-  addParams: [{ className: ROUTER, importPath: ANGULAR_ROUTER }],
```

Comment:
Regarding to line 17, I noticed that we are missing a comment with a file location. Is it optional? Or can we add `// projects/core/src/routing/facade/routing.service.ts` to this file to be consistent with other migrations.

Reply:
Thanks. Indeed, I overlooked it. Will add.